docs/source/index.rst

```diff
@@ -39,12 +39,11 @@ Index
 .. toctree::
    :maxdepth: 2
 
-   getting_started
-   node
-   broadcasting
-   bonding_and_delegation
+   quickstart
+   installation
+   overview
    transcoding
-   streamflow-public-testnet
```

Comment:
There are a lot of links to `streamflow-public-testnet`. Should we keep the page and just have a link back to the top level docs as content, or will removing the page automatically link people back to the toplevel?

Reply:
I *think* the link will break for a non-existent doc - linking back to the top level would be good.